.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+updates:
+- package-ecosystem: bundler
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
+  allow:
+  - dependency-type: direct
+  - dependency-type: indirect
+  ignore:
+  - dependency-name: sorbet-runtime
+    versions:
+    - 0.5.6371
```

Comment:
We're already running a newer version than this?

Reply:
Yeh, this will be the old version was ignored at some stage. We can probably go through and clear these all up.